include/odp/api/spec/init.h

```diff
@@ -32,6 +32,18 @@ extern "C" {
  * ODP instance ID.
  */
 
+/**
+ * Called from signal handler
+ */
+#define ODP_TERM_FROM_SIGH	0x1u
+
+/**
+ * Last standard flag for abnormal terminate
+ *
+ * An implementation may use this for adding its own flags after the standard flags.
+ */
+#define ODP_TERM_LAST_FLAG ODP_TERM_FROM_SIGH
```

Comment:
Please change to #define ODP_TERM_LAST_FLAG (1 << 32)ull else the logic will fail when new  generic ODP_TERM_... FLAG

Reply:
Idea is to keep updating the `ODP_TERM_LAST_FLAG` when new API-level flags are added.